### PR TITLE
Dutch Translation

### PR DESCRIPTION
**Pull request by David-NF**
This is the Dutch translation for you extension, the little icon is made by one of the other translators. I change the css to match the image and the link. Let me know when I need to fix or add something ;)

### CODE REVIEW

**Comment by ErnadoO** [comment]
Thanks :)

styles/prosilver/theme/nl/Thumbs.db ? :p

**Comment by David-NF** [comment]
I have no idea where that file came from, I'm using the desktop thing since a few days. So I forked your repo, cloned it into the Github Desktop program and then added the Dutch files and added a PR on the Github site, not sure where and when that file came up. Let me try to remove it...

**Comment by David-NF** [comment]
I think I removed it...

**Comment by ErnadoO** [comment]
Thumbs.db is a hidden file used to store thumbnail images. So, when you display a directory with images files (like theme/nl/), this file is automatically created by Windows ;)

**Comment by ErnadoO** [comment]
@David-NF Travis test failed, you need to remove spaces line 40 in nl/permissions_directory.php ;)

I also added a language key: https://www.phpbb.com/community/viewtopic.php?p=13998221#p13998221

File: styles/prosilver/theme/nl/directory.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+li.new {
+	background-position: 3px 50%;
+	background-repeat: no-repeat;
+	padding-left: 40px;
```
Comment (ErnadoO) [inline styles/prosilver/theme/nl/directory.css]: 35px ;)

Comment (David-NF) [inline styles/prosilver/theme/nl/directory.css]: Changed :)

**Comment by David-NF** [comment]
@ErnadoO Added new lang var, fixed the space line and changed css. Travis test is pending, how can I see the fail reasons?

**Comment by ErnadoO** [comment]
@David-NF You can view reports here: https://travis-ci.org/ErnadoO/ext-phpbb-directory/pull_requests ;)

File: language/nl/help_directory_flags.php [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+	'KI' => 'Kiribati',
+	'KM' => 'Comores',
+	'KN' => 'Saint Kitts and Nevis',
+	'KP' => 'Noord Korea',
+	'KR' => 'Zuid Korea',
+	'KW' => 'Koeweit',
+	'KY' => 'Kaaimaneilanden',
+	'KZ' => 'Kazachstan',
+	'LA' => 'Laos',
+	'LB' => 'Liban', //????
```
Comment (ErnadoO) [inline language/nl/help_directory_flags.php]: Libanon ?
http://nl.wikipedia.org/wiki/ISO_3166-2:LB ;)

Comment (David-NF) [inline language/nl/help_directory_flags.php]: I guess you're right, it looks like French it should be English ;) So you might want to check the English variable for this...

Comment (ErnadoO) [inline language/nl/help_directory_flags.php]: Indeed, my fault :D
I will fix it, thanks ;)

**Comment by ErnadoO** [comment]
Looks fine to me, i merge. Thank you very much ;)

**Comment by David-NF** [comment]
You're welcome glad I could help, I'm definitely going to use this extension when I'm going to 3.1 and when it's validated.

**Pull request merged**